Mathlib/NumberTheory/LSeries/HurwitzZetaEven.lean

```diff
@@ -395,3 +407,406 @@ lemma completedCosZeta₀_one_sub (a : UnitAddCircle) (s : ℂ) :
   rw [← completedHurwitzZetaEven₀_one_sub, sub_sub_cancel]
 
 end FEPair
+
+/-!
+## Differentiability and residues
+-/
+
+section FEPair
+
+/--
+The even Hurwitz completed zeta is differentiable away from `s = 0` and `s = 1` (and also at
+`s = 0` if `a ≠ 0`)
+-/
+lemma differentiableAt_completedHurwitzZetaEven
+    (a : UnitAddCircle) {s : ℂ} (hs : s ≠ 0 ∨ a ≠ 0) (hs' : s ≠ 1) :
+    DifferentiableAt ℂ (completedHurwitzZetaEven a) s := by
+  refine (((hurwitzEvenFEPair a).differentiableAt_Λ ?_ (Or.inl ?_)).comp s
+      (differentiableAt_id.div_const _)).div_const _
+  · rw [div_ne_zero_iff, eq_true_intro two_ne_zero, and_true]
+    refine Or.imp (by tauto) (fun ha ↦ ?_) hs
+    simp only [hurwitzEvenFEPair, eq_false_intro ha, if_false]
```

Comment:
```suggestion
  · simp only [ne_eq, div_eq_zero_iff, OfNat.ofNat_ne_zero, or_false]
    rcases hs with h | h
    · exact Or.inl h
    · simp only [hurwitzEvenFEPair, one_div, h, ↓reduceIte, or_true]
```
I think this reads a bit better (I find `eq_true_intro` and `eq_false_intro` somewhat suspicous).